<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>learn_widget</class>
 <widget class="QWidget" name="learn_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="2">
    <widget class="QPushButton" name="nextButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(
		spread:pad,
		x1:0, y1:0, x2:0, y2:1,
		stop:0 #2b1f0e
		stop:1 #8c6c26
	);

	color: white;
	font-weight: bold;
	border: 3px solid #d4af37;
	border-radius: 15px;
	padding: 10px;
}

QPushButton:hover{
	background-color: qlineargradient(
		spread:pad,
		x1:0, y1:0, x2:0, y2:1,
		stop:0 #3b2f1c,
		stop:1 #ad8c32
	);
}</string>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
    <widget class="QLabel" name="headerLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #ffd700;
	font-family: 'Georgia';
	font-size: 32px;
	font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="previousButton">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(
		spread:pad,
		x1:0, y1:0, x2:0, y2:1,
		stop:0 #2b1f0e
		stop:1 #8c6c26
	);

	color: white;
	font-weight: bold;
	border: 3px solid #d4af37;
	border-radius: 15px;
	padding: 10px;
}

QPushButton:hover{
	background-color: qlineargradient(
		spread:pad,
		x1:0, y1:0, x2:0, y2:1,
		stop:0 #3b2f1c,
		stop:1 #ad8c32
	);
}</string>
     </property>
     <property name="text">
      <string>prev</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
    <widget class="QLabel" name="instructionLabel">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: white;	/* makes the letters white */
	font-family: 'Georgia';
	font-size: 18px;
	background-color: #003b16; /* green color */
	border-radius: 20px; 
	border: 3px solid #ffd700;	/* yello border */
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
